src/components/PostCard.jsx

```diff
@@ -20,7 +33,8 @@ export default function PostCard({ post, onClick }) {
       </div>
       <div className="mt-2 px-1">
         <h4 className="text-base font-semibold text-dark">{post.title}</h4>
-        <p className="text-sm text-gray">{post.zip || "Unknown"}</p>
+        <p className="text-sm text-gray">
+          {cityState || post.zip || "Unknown"}
```

Comment:
Missing the closing tag for the `p`.

```suggestion
        <p className="text-sm text-gray">
          {cityState || post.zip || "Unknown"}
        </p>
```